### PR TITLE
Inline dropdown for role picker tweak

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 
Screen Shot 2019-04-26 at 1 30 16 PM

### CODE REVIEW

File: shared/common-adapters/box.desktop.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import type {Box2Props} from './box'
 
 class Box extends React.PureComponent<any> {
   render() {
-    const {forwardedRef, ...rest} = this.props
+    const {forwardedRef, inline, ...rest} = this.props
```
Comment (chrisnojima) [inline shared/common-adapters/box.desktop.js]: why is this here?

Comment (MarcoPolo) [reply shared/common-adapters/box.desktop.js]: whoops. I did that as a temp hack. will fix

Comment (MarcoPolo) [reply shared/common-adapters/box.desktop.js]: I added inline prop to button box, and styled by default forwards all props so I was getting an error in react because div was getting a boolean.
